datapreview.tex

```diff
@@ -66,6 +66,7 @@ \subsection{Data Preview 1}
 
 In both DP1 scenarios, the exact processing pipelines and data products are still under discussion.
 At minimum,  DP1 will deliver visit-level calibrated images and catalogs to enable initial studies of observational and instrumental effects. 
```

Comment:
At minimum, DP1 will deliver visit-level calibrated images and catalogs to enable initial studies of observational and instrumental effects.